presto-hive-functions/src/main/java/com/facebook/presto/hive/functions/scalar/HiveScalarFunction.java

```diff
@@ -48,7 +50,11 @@ public static HiveScalarFunction create(Class<?> cls, QualifiedObjectName name,
                 FunctionImplementationType.JAVA,
                 true,
                 true);
-        JavaScalarFunctionImplementation implementation = new HiveScalarFunctionImplementation(methodHandle);
+        InvocationConvention invocationConvention = new InvocationConvention(
```

Comment:
merge this change with the commit that introduces `HiveScalarFunctionImplementation`